pkg/server/handler.go

```diff
@@ -0,0 +1,101 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/interactor"
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+	"github.com/CESARBR/knot-cloud-storage/pkg/network"
+)
+
+const (
+	queueNameFogIn        = "storage-in"
```

Comment:
What do you think about renaming to "storage-thing-data". The usage of "in" seems redundant to me because the client only bind queues to an exchange in order to receive (incoming) data.

Reply:
Agree with you! Done!